core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -63,54 +63,70 @@ internal class CacheFallbackOperations
                 return;
             }
 
-            var rtItem = new MsalRefreshTokenCacheItem
-                (new Uri(authority).Host, clientId, resultWrapper.RefreshToken, resultWrapper.RawClientInfo);
-
-            tokenCacheAccessor.SaveRefreshToken(rtItem);
-
-            MsalAccountCacheItem accountCacheItem = new MsalAccountCacheItem
-                (new Uri(authority).Host, objectId, resultWrapper.RawClientInfo, null, displayableId, resultWrapper.Result.TenantId);
+            try
+            {
+                var rtItem = new MsalRefreshTokenCacheItem
+                    (new Uri(authority).Host, clientId, resultWrapper.RefreshToken, resultWrapper.RawClientInfo);
+                tokenCacheAccessor.SaveRefreshToken(rtItem);
 
-            tokenCacheAccessor.SaveAccount(accountCacheItem);
+                MsalAccountCacheItem accountCacheItem = new MsalAccountCacheItem
+                    (new Uri(authority).Host, objectId, resultWrapper.RawClientInfo, null, displayableId, resultWrapper.Result.TenantId);
+                tokenCacheAccessor.SaveAccount(accountCacheItem);
+            }
+            catch (Exception ex)
+            {
+                msg = "An error occurred while writing MSAL refresh token.";
```

Comment:
>"An error occurred while writing MSAL refresh token."; [](start = 22, length = 54)

Thinking the message should indicate that we are continuing with-out writing. Similar message for the other messages.

Reply:
The reason to do Suppressing exceptions in cache fall back logic  is to reduce impact of possible issues which might happen during cross library cache read write operations, so it is not part of library API , the intention is to provide more details for developers in logs to perform troubleshooting.